tools/cli/main.cpp

```diff
@@ -219,7 +251,11 @@ int main(int argc, const char **argv)
 		return run_command(executable_path.c_str(), extra_args,
 				   so_path.c_str(), nullptr);
 	} else if (program.is_subcommand_used("start")) {
+		#if __linux__
 		auto agent_path = install_path / "libbpftime-agent.so";
+		#elif __APPLE__
+		auto agent_path = install_path / "libbpftime-agent.dylib";
+		#endif
```

Comment:
```suggestion
#if __linux__
  #define AGENT_LIBRARY "libbpftime-agent.so"
#elif __APPLE__
  #define AGENT_LIBRARY "libbpftime-agent.dylib"
#endif 
auto agent_path = install_path / AGENT_LIBRARY;
```
The `AGENT_LIBRARY` definition would probably go higher up in the file and be reused below as well.

What do you think?

Reply:
yea, agreed, should have it in global space and replace where it is being used by the defined variable.